curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/62ff8e998d3e7eae14d6ae3b.md

```diff
@@ -9,9 +9,9 @@ dashedName: step-30
 
 You need to confirm that the user has read the terms and conditions.
 
-Add `label` element after the third `fieldset`, and a `input` element with `type` attribute of `checkbox` inside the newly added `label` element. Make this `input` element `required` because users should not sign up without reading the terms and conditions.
+Add a `label` element. Inside the newly created `label` element add an `input` element and set the `type` attribute to `checkbox`. Make this `input` element `required` so users can not sign up without agreeing to the terms and conditions.
 
-Don't forget to add an `id` attribute of `terms-and-conditions` for accessibility.
+Add an `id` and `for` attribute with the value &nbsp; &nbsp; &nbsp; &nbsp; `terms-and-conditions` to the elements for accessibility.
```

Comment:
```suggestion
Add an `id` and `for` attribute with the value `terms-and-conditions` to the elements for accessibility.
```

You have some extra whitespace here.

Reply:
I did that on purpose, the `terms and conditions` block was splitting into half.

Reply:
When the text will wrap to a new line is dependent on the screen width of the device. You can not predict that.

Here is how it looks on my screen.

![challenge-text](https://github.com/freeCodeCamp/freeCodeCamp/assets/28780271/199347e9-b906-4875-9eba-9ad00f93f3d8)

Please remove the spaces.

Reply:
Thanks, you are right :+1: